ui-tests/src/test/java/org/openmrs/reference/page/ClinicianFacingPatientDashboardPage.java

```diff
@@ -62,7 +62,7 @@
     private static final By LOCATION = By.id("w3");
     private static final By WHO_WHEN_WHERE = By.id("who-when-where");
     private static final By REQUEST_APPOINTMENT = By.linkText("Request Appointment");
-    private static final By APPOINTMENT_EDIT = By.cssSelector("div.info-section:nth-child(4) > div > a:nth-child(3)");
+    private static final By APPOINTMENT_LINK = By.cssSelector("a.right > i");
```

Comment:
For the naming: 
For code readability, APPOINTMENT_LINK makes more meaning than APPOINTMENT_EDIT since the element is pointing to a link of a feature.

For the attribute property:
One challenge with retrieving the element through `divs` is that once the `UI` changes comes in as I see the work from `MF- Squad ` they are doing, things get broken and the element will no longer be retrievable with this property. Putting that in mind and for future software usage, I opted rather to set the property  to use `By.cssSelector("a.right > i");`